io.github.TTimo.GtkRadiant.json

```diff
@@ -0,0 +1,220 @@
+          "path": "patches/gtkglext-1.2.0/01_fix_fontcache_nullderef.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/02_fix_gtk-2.20_deprecated_symbols.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/libGL.so.1.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/03_gdkglext-config-h-installation-path.diff"
+        },
+        {
+          "type": "patch",
+          "path": "patches/gtkglext-1.2.0/glibc2.27-ftbfs.diff"
+        }
+      ]
+    },
+    {
+      "name": "gtkradiant",
+      "no-autogen": true,
+      "sources": [
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/ETPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/ETPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/JAPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/JAPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/Q1Pack/trunk",
+          "revision": "144",
+          "dest": "install/installs/Q1Pack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/Q2Pack/trunk",
+          "revision": "144",
+          "dest": "install/installs/Q2Pack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/Q3Pack/trunk",
+          "revision": "144",
+          "dest": "install/installs/Q3Pack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/QLPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/QLPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/QuetooPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/QuetooPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/STVEFPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/STVEFPack"
+        },
+        {
+          "type": "svn",
+          "url": "https://github.com/Unvanquished/unvanquished-mapeditor-support.git/trunk/build/gtkradiant/",
+          "revision": "207",
+          "dest": "install/installs/UnvanquishedPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/UrTPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/UrTPack"
+        },
+        {
+          "type": "svn",
+          "url": "svn://svn.icculus.org/gtkradiant-gamepacks/WolfPack/trunk",
+          "revision": "144",
+          "dest": "install/installs/WolfPack"
+        },
+        {
```

Comment:
I'd move this source at the top of list since it's the main one. It doesn't change the build, but that's a good practice.